src/audio/data.c

```diff
@@ -612,8 +612,10 @@ u16 gHeadsetPanQuantization[0x10] = {
     0x40, 0x40, 0x30, 0x30, 0x20, 0x20, 0x10, 0, 0, 0,
 };
 #elif !defined(VERSION_SH)
+#ifndef DISABLE_HEADSET_STEREO_EFFECTS
 u16 gHeadsetPanQuantization[10] = { 0x40, 0x30, 0x20, 0x10, 0, 0, 0, 0, 0, 0 };
 #endif
+#endif
```

Comment:
```suggestion
#elif !defined(VERSION_SH) && !defined(DISABLE_HEADSET_STEREO_EFFECTS)
u16 gHeadsetPanQuantization[10] = { 0x40, 0x30, 0x20, 0x10, 0, 0, 0, 0, 0, 0 };
#endif
```

Reply:
hmm i guess because of the other headers this might be a bigger change than worth it. up to you. resolve if you dont want to do this lmao

Reply:
I noticed this too when looking at it recently, I think prefer the elif

Reply:
same